Apps/GemDigest/docker-compose.yml

```
@@ -0,0 +1,97 @@
+  # Supported CPU architectures for the application
+  architectures:
+    - amd64
+    - arm64
+
+  # Main service of the application
+  main: big-bear-gem_digest_bot
+
+  description:
+    # Description in English
+    en_us: Simplify your reads, amplify your time! 📚✨ GemDigest is a Telegram bot that summarizes articles from any website into bite-sized nuggets, saving you time and energy for what matters most. 📱👋
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)